commandLineGame/outdoor_activity_generator.js

```diff
@@ -0,0 +1,269 @@
+    }
+  }
+};
+
+
+function inactiveTemp65OrLess(userInput, compChoice) {
+  if (userInput === compChoice) {
+    return `${govCuomo()}`;
+  }
+  if (userInput === 'rock') {
+    if (compChoice === 'paper') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend to visit an ${notPhysActive[0]}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's ${temp}°F, I recommend to watch a ${notPhysActive[1]}.`;
+    }
+  }
+
+  if (userInput === 'paper') {
+    if (compChoice === 'scissors') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend: ${notPhysActive[2]} for your ${aloneOrGroup}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's  ${temp}°F, I recommend to: ${notPhysActive[3]}.`;
+    }
+  }
+
+  if (userInput === 'scissors') {
+    if (compChoice === 'rock') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend to relax while ${notPhysActive[4]}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's ${temp}°F, I recommend you ${notPhysActive[5]}.`;
+    }
+  }
+};
+
+function inactiveTemp66OrMore(userInput, compChoice) {
+  if (userInput === compChoice) {
+    return `${govCuomo()}`;
+  }
+  if (userInput === 'rock') {
+    if (compChoice === 'paper') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend to host an ${notPhysActive[6]}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's ${temp}°F, I recommend to host an ${notPhysActive[7]}.`;
+    }
+  }
+
+  if (userInput === 'paper') {
+    if (compChoice === 'scissors') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend a ${notPhysActive[8]}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's  ${temp}°F, I recommend to go ${notPhysActive[9]}.`;
+    }
+  }
+
+  if (userInput === 'scissors') {
+    if (compChoice === 'rock') {
+      return `I won! ${name}, because it's ${temp}°F, I recommend to ${notPhysActive[10]}.`;
+    } else {
+      return `Congrats, you won! ${name}, because it's ${temp}°F, I recommend to ${notPhysActive[11]}.`;
+    }
+  }
+};
+
+
+function govCuomo(){
+console.log("We tied! SURPRISE! Gov. Cuomo is giving two free tickets to the Buffalo Bills game today at Bills Stadium!")
+if (readLineSync.keyInYN("Would you like to reserve your tickets?")) {
+  console.log("Great! Make sure your latest PCR test results are within 72 hours prior to today's date.")
+  if(readLineSync.keyInYN("Do your recent test results meet the 72 hour window?")){
+    console.log("Great! Ticket's are yours!")
+    endGame();
+  }
+  else{
+    console.log("That's OK. Let's play another round to decide what activity you'll to today.")
+    rockPaperScissors();
+  }
+}else{
+  console.log("Since you don't want to reserve available tickets, lets play another round of rock, paper, scissors till I can recommend an activity.")
+  rockPaperScissors();
+}
+endGame();
```

Comment:
This is getting called at the end of all the conditions and is therefor causing me to always end the game instead of having the option to play again.